Makefile

```diff
@@ -8,7 +8,7 @@ TESTS = test/lib/* \
 all: lint test
```

Comment:
We need to copy the whole Makefile from https://github.com/catberry/catberry/blob/develop/Makefile and replace `SOURCES` and `TESTS` values.